src/schema/server_settings_schema.py

```diff
@@ -208,6 +248,12 @@
         "href": "general-settings",
         "settings": GENERAL_OPTIONS,
     },
+    {
+        "title": "Magic Mode",
+        "description": "Generate an entire game from scratch.",
+        "href": "magic-mode",
+        "settings": [],
```

Comment:
The UI doesn't have any options yet -- I think the second pass of this will work that in once we know what those options even are (story title? story url?)